ui/src/ensembler/components/table/ListEnsemblingJobsForEnsemblerTable.js

```diff
@@ -0,0 +1,124 @@
+import React, { Fragment, useEffect, useState } from "react";
+import {
+  EuiCallOut,
+  EuiBasicTable,
+  EuiText,
+} from "@elastic/eui";
+import { DeploymentStatusHealth } from "../../../components/status_health/DeploymentStatusHealth";
+import { JobStatus } from "../../../services/job/JobStatus";
+import { useConfig } from "../../../config";
+import { useTuringApi } from "../../../hooks/useTuringApi";
+
+
+export const ListEnsemblingJobsForEnsemblerTable = ({
+  projectID,
+  ensemblerID,
+  setDisablePopup,
+  disablePopup
+}) => {
+  const [results, setResults] = useState({ inactiveItems: [], activeItems:[], totalInactiveCount: 0, totalActiveCount:0 });
+
+  const {
+    appConfig: {
+      tables: { defaultTextSize },
+    },
+  } = useConfig();
+
+  const [{ data, isLoaded, error }] = useTuringApi(
+    `/projects/${projectID}/jobs?ensembler_id=${ensemblerID}`,
+    []
+  )
+
+  useEffect(() => {
+    if (isLoaded && !error) {
+      let inactiveItems = data.results.filter((item) => item.status === 'failed_submission' || item.status==='failed_building' || item.status==='failed' || item.status==='completed');
+      let activeItems = data.results.filter((item) => item.status !== 'failed_submission' && item.status!=='failed_building' && item.status!=='failed' && item.status!=='completed');
+      setResults({
+        inactiveItems: inactiveItems ,
+        activeItems: activeItems ,
+        totalInactiveCount: inactiveItems.length,
+        totalActiveCount : activeItems.length
+      });
+      if (activeItems.length > 0){
+        setDisablePopup(true)
```

Comment:
Do we need to set this from here? Instead, can we just set the results from here and use another `useEffect` hook to enable the popup according to the results?